### DISCUSSION TITLE
OAuth isn't workign anymore

### DISCUSSION THREAD

**Discussion by paustian** [zikula/core#5233] [General]
So, I am up for a new place to have discussions. I noticed that when I upgraded to Ubuntu 22 and PHP 8.0, the OAuth module broke. I was able to replicate this on my test server. I am happy to look into this, but have some questions....

I cannot get to the main page for OAuth because the route is broken. Any suggestions on where to start on this? That would suggest to me something is funky in  Menu/ExtensionMenu.php. When I run a debugger on it, I see this error:

`Unable to generate a URL for the named route "zikulaoauthmodule_mapping_list" as such route does not exist.`

The code its breaking on is line 64 in Menu/ExtensionMenu.php:
`$menu->addChild('Mapping list', [
                'route' => 'zikulaoauthmodule_mapping_list',
            ])->setAttribute('icon', 'fas fa-list');`

Thanks.

**Comment by Guite** [comment]
Hey @paustian,

which versions of Zikula core and the OAuth module do you use?
